plugins/contiv/plugin_impl_contiv.go

```diff
@@ -626,17 +626,17 @@ func appendIfMissing(slice []string, s string) []string {
 
 // getIPAMConfig populates the Config struct with the calculated subnets
 func getIPAMConfig(config *Config) *Config {
-	_, contivNetwork, _ := net.ParseCIDR(config.IPAMConfig.ContivCIDR)
+	_, contivNetwork, _ := net.ParseCIDR("10.1.0.0/14")
```

Comment:
the param should be used instead of constant

Reply:
Right, this went in from my testing.